model_databank/views.py

```diff
@@ -87,11 +89,35 @@ def form_valid(self, form):
                                       "processed soon."))
         return super(ModelUploadFormView, self).form_valid(form)
 
+    # N.B. the methods below this point are needed because the
+    # RoleRequiredRemoteMixin overrides the dispatch() method
+    def handle_not_logged_in(self):
+        return redirect_to_login(
+            self.request.get_full_path(), settings.LOGIN_URL,
+            self.get_redirect_field_name())
 
-class ModelDownloadView(DetailView):
+    def get_redirect_field_name(self):
+        return REDIRECT_FIELD_NAME
```

Comment:
Ja, hoeft niet per se, is overgenomen uit een andere class waarin ie overridable was, maar dat doe ik hier niet.